stable/jenkins/templates/config.yaml

```diff
@@ -170,6 +170,9 @@ data:
     <?xml version='1.1' encoding='UTF-8'?>
     <jenkins.model.JenkinsLocationConfiguration>
       <adminAddress>{{ default "" .Values.Master.JenkinsAdminEmail }}</adminAddress>
+{{- if .Values.Master.JenkinsUrl }}
```

Comment:
Could another option be to add a `forceHttps` flag rather than specify the whole URL. Then the following line could change from:

```
{{- if .Values.Master.Ingress.TLS }}
```

to:

```
{{- if .Values.Master.Ingress.TLS or .Values.Master.forceHttps }}
```

and the rest remain as it currently is?

Reply:
I think its fine to leave the if as is.